docs/core/config/auth/databases/ldap.md

```diff
@@ -202,12 +219,10 @@ This sections describes the configuration common to LDAP [[link,passdb]] and
 
 ### Connecting
 
-There are two alternative ways to specify what LDAP server(s) to connect to:
+The LDAP server(s) endpoints must be specified as ldap URIs:
 
-* `hosts`: A space separated list of LDAP hosts to connect to. You can also
-  use `host:port` syntax to use different ports.
-* `uris`: A space separated list of LDAP URIs to connect to. This isn't
-  supported by all LDAP libraries. The URIs are in syntax
+* `ldap_uris`: A space separated list of LDAP URIs to connect to.
+  The URIs are in syntax
   `protocol://host:port`. For example `ldap://localhost` or
   `ldaps://secure.domain.org`.
```

Comment:
This explanation belongs to the ldap_uris setting. If wanted, the setting could be included here. But in any case not duplicated text.